### PR TITLE
Feature2 Register Feature Added and Login Session Handled

### PR DESCRIPTION
**Pull request by SivaSS-crypto**
- [ x] Register Feature Added 
- [x] Validation for User details added
- [x]Login session added

### CODE REVIEW

File: src/main/java/in/siva/service/UserManagement.java [hunk 1]
```diff
@@ -58,5 +58,22 @@ public static boolean loginValidation(String userName, String userPassword) {
...
 
+/**
+	 * Register with user details Add the user Details in ArrayList
+	 * 
+	 * @param list //Details of user
+	 */
+	public static boolean registerDetails(User user) {
+		boolean register = false;
+		// Condition for valid details
+		if (ValidUser.isValidUser(user)) {
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/siva/service/UserManagement.java:69]: UserValidator => isValidUser(user)

**Review by ghost — CHANGES_REQUESTED** [review]
Review changes commented

**Review by ghost — APPROVED** [review]
LGTM

File: src/main/java/in/siva/service/UserManagement.java (hunk 1 above)
Comment (SivaSS-crypto) [reply src/main/java/in/siva/service/UserManagement.java:69]: Changed the class name sir

**Pull request merged**